src/Data/Nat/Properties.agda

```diff
@@ -670,6 +676,25 @@ m+n≮m m n = subst (_≮ m) (+-comm n m) (m+n≮n n m)
   n * o + m * (n * o) ≡⟨⟩
   suc m * (n * o)     ∎
 
+m*[1+n] :  ∀ m n → m * (suc n) ≡ m + m * n
+m*[1+n] m n = begin-equality
+  m * (1 + n)     ≡⟨ *-distribˡ-+ m 1 n ⟩
+  m * 1 + m * n   ≡⟨ cong (_+ (m * n)) (*-identityʳ  m) ⟩
+  m + m * n       ∎
+
+m>1⇒m*n≢1 :  ∀ {m n} → m > 1 → m * n ≢ 1
+m>1⇒m*n≢1 {m} {0}     _   m*0≡1  =  0≢1+n 0≡1
+                                   where
+                                   0≡1 = trans (sym (*-zeroʳ m)) m*0≡1
+
```

Comment:
`where` blocks should be indented by two spaces, cf. [the style guide](https://github.com/agda/agda-stdlib/blob/master/notes/style-guide.md#indentation)

Reply:
Done